torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -52,6 +52,11 @@ inline Tensor relu(Tensor& input, const ReLUOptions& options) {
   }
 }
 
+inline Tensor relu6(Tensor& input, const ReLU6Options& options) {
```

Comment:
similarly here: `options` can be optional, and we should add a test for this too.